test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/ObjectResultTests.cs

```diff
@@ -94,7 +98,7 @@ public void ObjectResult_Create_CallsContentResult_InitializesValue()
         {
             // Arrange
             var input = "testInput";
-            var actionContext = CreateMockActionContext();
+            var actionContext = CreateMockActionContext(new IOutputFormatter[] { new TextPlainFormatter(), new JsonOutputFormatter() });
```

Comment:
This is very noisy. `CreateMockActionContext` should remain simple (perhaps `CreateDefaultMockActionContext`) and add an overload/another method when you want to customize